Dockerfile

```diff
@@ -51,6 +50,7 @@ ARG TOMCAT_VERSION
 ENV TOMCAT_VERSION=${TOMCAT_VERSION} \
     JAVA_CACERTS=$JAVA_HOME/jre/lib/security/cacerts \
 	CATALINA_BASE=/opt/apache-tomcat \
+    TRUSTSTORE=/opt/apache-tomcat/truststore.jks \
```

Comment:
Yes. This is partof the major shift away from privilege mode to user mode. Java is run as `plantuml:plantuml` anyway, so it can reside outside `/etc` as well.

Reply:
https://stackoverflow.com/questions/4097432/which-is-the-default-location-for-keystore-truststore-of-java-applications

This SO sounds so that this is no security issue